lib/src/widgets/html_editor_widget_web.dart

```diff
@@ -531,55 +531,65 @@ class _HtmlEditorWidgetWebState extends State<HtmlEditorWidget> {
-                  child: FutureBuilder<bool>(
-                      future: summernoteInit,
-                      builder: (context, snapshot) {
-                        if (snapshot.hasData) {
-                          return HtmlElementView(
-                            viewType: createdViewId,
-                          );
-                        } else {
-                          return Container(
-                              height: widget.htmlEditorOptions.autoAdjustHeight
-                                  ? actualHeight
-                                  : widget.otherOptions.height);
-                        }
-                      }))),
-          widget.htmlToolbarOptions.toolbarPosition ==
-                  ToolbarPosition.belowEditor
-              ? ToolbarWidget(
-                  key: toolbarKey,
-                  controller: widget.controller,
-                  htmlToolbarOptions: widget.htmlToolbarOptions,
-                  callbacks: widget.callbacks)
-              : Container(height: 0, width: 0),
-        ],
-      ),
-    );
+    return FutureBuilder<void>(
+        future: memoizer.runOnce(() => initSummernote(context)),
+        builder: (context, state) {
+          if (state.hasError) return ErrorWidget(state.error!);
+          if (state.connectionState != ConnectionState.done) {
+            return SizedBox.shrink();
+          }
+          return Container(
+            height: widget.htmlEditorOptions.autoAdjustHeight
+                ? actualHeight
+                : widget.otherOptions.height,
+            child: Column(
+              children: <Widget>[
+                widget.htmlToolbarOptions.toolbarPosition ==
+                        ToolbarPosition.aboveEditor
+                    ? ToolbarWidget(
+                        key: toolbarKey,
+                        controller: widget.controller,
+                        htmlToolbarOptions: widget.htmlToolbarOptions,
+                        callbacks: widget.callbacks)
+                    : Container(height: 0, width: 0),
+                Expanded(
+                    child: Directionality(
+                        textDirection: TextDirection.ltr,
+                        child: FutureBuilder<bool>(
+                            future: summernoteInit,
+                            builder: (context, snapshot) {
+                              if (snapshot.hasData) {
+                                return HtmlElementView(
+                                  viewType: createdViewId,
+                                );
+                              } else {
+                                return Container(
+                                    height: widget
+                                            .htmlEditorOptions.autoAdjustHeight
+                                        ? actualHeight
+                                        : widget.otherOptions.height);
+                              }
+                            }))),
```

Comment:
Now that you have added a `FutureBuilder` surrounding the entirety of the build method, do we need this secondary `FutureBuilder`, as well as the `summernoteInit` variable? It seems to me we can get rid of this.

Reply:
Sounds good, will re-review when you get a chance to remove it :)

Reply:
@tneotia should be ok now